interfaces/apparmor/backend.go

```diff
@@ -0,0 +1,149 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+// Package apparmor implements integration between snappy and
+// ubuntu-core-launcher around apparmor.
+//
+// Snappy creates apparmor profiles for each application (for each snap)
+// present in the system.  Upon each execution of ubuntu-core-launcher, the
+// profile is attached to the running process. Prior to that the profile must
```

Comment:
Minor nitpick-- the profile isn't attached to the process so much as the process is launched under the profile.